components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/claims/impl/DefaultClaimHandler.java

```diff
@@ -1168,6 +1168,13 @@ protected void handleApplicationRolesForLocalUser(StepConfig stepConfig, Authent
                                                     Map<String, String> allLocalClaims)
             throws FrameworkException {
 
+        /**
```

Comment:
Comments in the form of Java doc comments?
Can we have multiline comments in this format
```
/*
*/
```

Reply:
Addressed in 2b653fdda222c2da9375ffa7d6dc8ccb8f0ca329